felayout_t3kit/dev/styles/main/contentElements/heroImage.less

```diff
@@ -41,20 +41,21 @@
 }
 
 .hero-image__caption {
-    padding: 20px 60px;
+    padding: 15px;
     color: @main-body-bg;
     overflow: hidden;
     -webkit-font-smoothing: antialiased;
 }
 
-.hero-image__caption .hero-image__caption-header {
+.hero-image__caption-header {
     color: @main-body-bg;
-    font-style: normal;
+    margin-top: 5px;
+    font-size: @font-size-h2 * 0.86;
 }
 
-.hero-image__caption .hero-image__caption-subheader {
+.hero-image__caption-subheader {
     color: @main-body-bg;
-    font-style: italic;
+    font-size: @font-size-h3 * 0.77;
 }
 
 .hero-image__caption-link .btn {
```

Comment:
tag `a` should have its own unique class. For example, in this case, it could be `hero-image__caption-link`